tests/integration/targets/iso_customize/files/ks.cfg

```diff
@@ -0,0 +1,42 @@
+#Generated by Kickstart Configurator
+#platform=x86
+
+#System language
+lang en_US
+#Language modules to install
+langsupport en_US
+#System keyboard
+keyboard us
+#System mouse
+mouse
+#System timezone
+timezone America/New_York
+#Root password
+rootpw --iscrypted $1$VjstxyI6$8leT50BkNzJ9t5Z.GAAct0
+#Initial user
+user vmware --fullname "vmware" --iscrypted --password $1$qQpKmzVk$jcBmfWYLIDKA8FPG5gwfF1
+#Reboot after installation
+reboot
+#Use text mode install
+text
+#Install OS instead of upgrade
+install
+#Use CDROM installation media
+cdrom
+#System bootloader configuration
+bootloader --location=mbr 
+#Clear the Master Boot Record
+zerombr yes
+#Partition clearing information
+clearpart --all --initlabel 
+#System authorization infomation
+auth  --useshadow  --enablemd5 
+#Network information
+network --bootproto=dhcp --device=eth0
+#Firewall configuration
+firewall --disabled --ssh 
+#Do not configure the X Window System
+skipx
+#install packages
+%packages
+openssh-server
```

Comment:
Since it is not possible to add comments to the binary files below: could you please remove them, and instead add some tasks at the beginning of the tests which create them?

Reply:
Thanks. The file is deleted.